### ISSUE TITLE
Inference-time dropout randomness is not disabled (F.dropout2d missing training=self.training)

### ISSUE THREAD

**Issue by BIOBRICK** [Bin-Chen-Lab/GPS#4]
Very Grateful for this open-source GPS tools! However, I noticed that  GPS_runPredProfile predictions are stochastic and the released *_strong.npz cannot be reproduced. 
I think that problem might be blamed: 
GPS4Drugs/code/model.py:24 and GPS4Drugs/code/model.py:28
h = F.dropout2d(h, p=self.dropout_rate)   # training not passed -> defaults to True
GPS4Drugs/code/apply_RCL.py:98 (model.eval() is called, but it does not affect functional dropout)
Same pattern in RCL/code/main_multi.py:203 (evaluate() also relies on model.eval())

I'm tried turn off the dropout randomness by using:
# model.py
h = F.dropout(h, p=self.dropout_rate, training=self.training)

at both dropout sites (lines 24 and 28), so that model.eval() actually disables dropout. Then regenerate the *_strong.npz libraries deterministically. (But ! If stochastic predictions are desired, expose an explicit MC-dropout option instead.)

So I'm not quite sure how this part is actually designed and the method in the original paper doesn't include such details so it's confusing for me to use GPS in my own data.

Any help will be gratefull! Thanks a lot!
